src/fillTank.test.js

```diff
@@ -1,11 +1,118 @@
 'use strict';
 
+const { fillTank } = require('./fillTank');
```

Comment:
It's generally a good practice to place all imports at the top of the file to improve readability and maintainability.